implicit-cordapp-upgrades/rpc-client/src/main/kotlin/net/obligation/rpcClient/RpcClient.kt

```diff
@@ -1,15 +1,10 @@
 package net.obligation.rpcClient
```

Comment:
While you're here, can you fix the package name - it shouldn't have a capital letter.